spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -872,6 +890,7 @@ def main(argv: Sequence[str]):
                 fname_input=fname_mask,
                 fname_label=lesion_obj.fname_label,
                 fname_sc=fname_sc,
+                orig_orientation=Image(fname_mask).orientation,
```

Comment:
Do we need to pass `orig_orientation` as a parameter? (I notice that at the start of the QC function, we have access to `Image(fname_mask)`, so could we just grab it then?)

Reply:
Good point! You are right! Fixed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4597/commits/71a102ac726d0336a9e3ed9c230a34c9edbc08b7